client/disputes/new-evidence/index.tsx

```diff
@@ -0,0 +1,83 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import React, { useState } from 'react';
+
+/**
+ * Internal dependencies.
+ */
+import { Stepper, StepperIndicator, useStepperContext } from '../stepper';
+import { Button } from 'wcpay/components/wp-components-wrapped';
+import '../style.scss';
+
+interface StepProps {
+	name: string;
+}
+
+const StepOne: React.FC< StepProps > = () => {
+	const { nextStep } = useStepperContext();
+	return (
+		<div>
+			<h2>General evidence</h2>
+			<p>Provide general evidence for your dispute.</p>
+			<Button variant="primary" onClick={ nextStep }>
+				Next
+			</Button>
+		</div>
+	);
+};
+
+const StepTwo: React.FC< StepProps > = () => {
+	const { nextStep, prevStep } = useStepperContext();
+	return (
+		<div>
+			<h2>Shipping information</h2>
+			<p>Provide shipping details if applicable.</p>
+			<Button variant="secondary" onClick={ prevStep }>
+				Previous
+			</Button>
+			<Button variant="primary" onClick={ nextStep }>
+				Next
+			</Button>
+		</div>
+	);
+};
+
+const StepThree: React.FC< StepProps > = () => {
+	const { prevStep } = useStepperContext();
+	return (
+		<div>
+			<h2>Review</h2>
+			<p>Review your information before submitting.</p>
+			<Button variant="secondary" onClick={ prevStep }>
+				Previous
+			</Button>
+			<Button variant="primary" disabled>
+				Submit
+			</Button>
+		</div>
+	);
+};
+
+export default () => {
+	const steps = [ 'General evidence', 'Shipping information', 'Review' ];
```

Comment:
It won't matter _too_ much in this implementation, but a slight opportunity to ensure that the `StepperIndicator` doesn't have to re-render each time this component renders.Moving `steps` outside of this component's body ensures that the array has a stable reference across re-renders.

```suggestion

const steps = [ 'General evidence', 'Shipping information', 'Review' ];

export default () => {
```

Reply:
thanks, will keep that in mind 0deacb8541b60d0ab66e4434b5c624b27bb218ab.